src/View/Admin/WooTab.php

```diff
@@ -498,42 +622,38 @@ private function get_store_information_settings() {
 	}
 
 	/**
-	 * Get the customer marketing settings.
+	 * Show the welcome screen if it's not connected.
 	 *
-	 * @since  2019-03-08
-	 * @author Zach Owen <zach@webdevstudios>
+	 * @since  2022-06-23
+	 * @author Biplav Subedi <biplav.subedi@webdevstudios>
 	 * @return array
```

Comment:
Another small spot for the since, and Zach shouldn't have done it with a date previously either.